src/project.ts

```diff
@@ -79,7 +84,12 @@ export async function readProject(
   projectDirectoryPath: string,
 ): Promise<Project> {
   const repositoryUrl = await getRepositoryHttpsUrl(projectDirectoryPath);
-  const rootPackage = await readPackage(projectDirectoryPath);
+  const tagNames = await getTagNames(projectDirectoryPath);
+  const rootPackage = await readMonorepoRootPackage({
```

Comment:
Nit: In this context, this "monorepo root package" might be a standalone package, rather than the root of a monorepo. Maybe we should drop "monorepo" from this function name so that isn't misunderstood.

Reply:
Oh wait, nevermind, this is only for the monorepo case. So maybe it's the description of this function that should change

Reply:
Just to be clear you're saying I should drop mention of "polyrepo" from the description for now, yeah?

Reply:
Yeah exactly; the part about how it handles polyrepo packages isn't accurate anymore because of these changes, so it can be dropped

Reply:
Fixed in [8f7703f](https://github.com/MetaMask/create-release-branch/pull/23/commits/8f7703f6ab475d5368f0b4404866163e13aac6f8).